app/src/main/java/org/liberty/android/fantastischmemo/entity/Deck.java

```diff
@@ -3,6 +3,7 @@
 import com.j256.ormlite.field.DataType;
 import com.j256.ormlite.field.DatabaseField;
 import com.j256.ormlite.table.DatabaseTable;
+import java.util.*;
```

Comment:
java import statements should be the last one according to the AOSP coding style. Also since you are importing .* remove the .Date that is there since it is included in .*